src/components/Header.jsx

```diff
@@ -1,19 +1,57 @@
-import { useNavigate } from 'react-router-dom';
-import logo from '../assets/0012p000041aQ4LAAU.png'
+import { useNavigate } from "react-router-dom";
+import logo from "../assets/0012p000041aQ4LAAU.png";
+import logOutIcon from "../assets/ic_exit_to_app_24px.png";
+import EmployeesIcon from "../assets/ic_supervisor_account_24px.png";
+import InventoryIcon from "../assets/BooksOverview.png";
+import AdminIcon from "../assets/ic_account_box_24px_admin.png";
+import { useContext } from "react";
+import AuthContext from "../store/auth-context";
 
+// TODO - fix css
 function Header() {
+  const auth = useContext(AuthContext);
   const navigate = useNavigate();
 
-    const redirectMainHandler = () => {
-      navigate('main', {replace: true});
-    }
+  const logoutHandler = () => {
+    auth.logout();
+    navigate("/", { replace: true });
+  };
+
+  const employeesHandler = () => {
+    navigate("/employees", { replace: true });
+  };
+
+  const redirectMainHandler = () => {
+    navigate("main", { replace: true });
+  };
 
   return (
     <div className="header">
-      <button><img src={logo} alt="Header" className="header-img" onClick={redirectMainHandler}/></button>
+      <nav className="navbar">
+        <img src={AdminIcon} alt="Admin" />
+        <span>Voornaam Achternaam</span>
+        <img className="static" src={InventoryIcon} alt="inventaris" />
+        <button>
+          <img
+            src={EmployeesIcon}
+            alt="Werknemers"
+            onClick={employeesHandler}
+          />
+        </button>
+        <button>
+          <img src={logOutIcon} alt="log out" onClick={logoutHandler} />
+        </button>
+      </nav>
+      <button>
+        <img
+          src={logo}
+          alt="Header"
+          className="header-img"
+          onClick={redirectMainHandler}
+        />
+      </button>
```

Comment:
Hoe zorg je nu voor het verschil tussen user en admin?

Reply:
In SortedTable zit een if statement, waarbij er eerst gekeken wordt of het een admin is ja of nee. Het is niet op een hele nette manier, want er zit dubbele code in, maar voor nu werkt het.